nav/support/sim.py

```
@@ -0,0 +1,480 @@
+                             R_min: float,
+                             R_max: float,
+                             crater_power_law_exponent: float,
+                             crater_relief_scale: float,
+                             work_center_v: float,
+                             work_center_u: float,
+                             aa_scale: int,
+                             lighting_angle: float,
+                             phase_angle: float,
+                             *,
+                             ellipse_mask: NDArrayFloatType,
+                             v_rot: NDArrayFloatType,
```

Ruff (0.14.4)

Unused function argument: `v_rot`

(ARG001)